templates/main.ejs

```diff
@@ -37,21 +37,33 @@
             const theme = getCookie('data-theme') === 'dark' ? 'light' : 'dark';
             setTheme(theme);
         }
+
+        function navigateTo(url) {
+            window.location.href = url;
+        }
+    </script>
+    <script>
+      function setResponseCookies(response) {
+          for (const [key, value] of Object.entries(response.set_variables)) {
+            document.cookie=`${key}=${value}`;
+          }
+        }
     </script>
   </head>
   <body>
     <main id="main">
         <div class="header">
           <button class="header-button" hx-get="/menu" hx-target="#content" hx-swap="innerHTML">MAIN MENU</button>
           <h2><%= header %></h2>
-          <button class="header-button" hx-get="/stats" hx-target="#content" hx-swap="innerHTML">STATS</button>
+          <button class="header-button" onclick="navigateTo('/stats')">STATS</button>
```

Comment:
This change is imo important, as I believe it is better to have the stats page as a separate one. 
The Single-Page Architecture, I am not sure if it works very well with ejs. With React, it'd be great, with ejs, not so sure.